### DISCUSSION TITLE
Verified Student but don’t receive any benebits and e-mail

### DISCUSSION THREAD

**Discussion by ISX901006** [community/community#69374] [GitHub Education] [answered]
### Hi everyone,

I am approved on 30th Sep as current student, and the buttom is purple now. 
However, I don't get any e-mail and benefits. I tried to reverify my identity, but the system shows that "You were last verified as a student on . It is not necessary for you to re verify at this time".
How can I solve it?

**Comment by mantasu** [comment, marked as answer]
I think you're having the same issue as some other people. See the answer [here](https://github.com/orgs/community/discussions/69264)

Basically, you may still need to wait for a few more days as it seems the staff is super busy currently. It doesn't hurt to open a [ticket](https://support.github.com/request/education), just remember to close it once your benefits arrive.
